setup.cfg

```diff
@@ -36,7 +36,7 @@ tests =
     invenio-search[opensearch2]>=2.0.0
     pytest-invenio>=2.1.0,<3.0.0
     sphinx>=4.5.0
-    invenio-app-rdm>=12.0.0,<14.0.0
+    invenio-app-rdm>=12.0.7,<14.0.0
```

Comment:
v12.0.7 includes the group service result resolver, so putting this as minimum allows us to kick out some config